pkg/controllers/dashboard/hostedkubernetecharts/controller.go

```diff
@@ -0,0 +1,106 @@
+package hostedkubernetecharts
```

Comment:
nit: It seems to be intentional. Perhaps, `hostedkubecharts` or `hostedcharts`? I might have missed the conversations leading to this.

Reply:
This was just a random name I gave for the package name, I am up for any good suggestion.

Reply:
I would personally prefer `hostedkubecharts` or `hostedclustercharts` if we want to keep it similar.